### DISCUSSION TITLE
Telemetry link dead

### DISCUSSION THREAD

**Discussion by theHacker** [requarks/wiki#7136] [Error / Bug Report] [answered]
Hi,

just trying out Wiki.js and reading through the docs.
https://docs.requarks.io/telemetry links to https://wiki.js.org/telemetry, however this is a 404 page.

> The aggregated anonymized telemetry stats are public and available at all times [here](https://wiki.js.org/telemetry).

**Comment by NGPixel** [comment, marked as answer]
The dashboards are no longer available and telemetry will be removed completely in the upcoming v3.

**Reply by benjamincarmel** [reply]
Why remove telemetry wouldn't it be useful to continue to collect data?
